src/core/MIGRATION.md

```diff
@@ -1115,20 +1115,23 @@ class MyPlugin {
   }
 ```
 
-### Mock core services in tests
-Core services already provide mocks to simplify testing and make sure plugins always rely on valid public contracts.
+### Mock new platform services in tests
+
+#### Writing mocks for your plugin
+Core services already provide mocks to simplify testing and make sure plugins always rely on valid public contracts:
 ```typescript
 // my_plugin/server/plugin.test.ts
-Import { configServiceMock } from 'src/core/server/mocks.ts'
+import { configServiceMock } from 'src/core/server/mocks';
```

Comment:
Is it worth mentioning this type of mocking? 

https://github.com/elastic/kibana/pull/42159/files#diff-a1b1987973ae75e774aa6e15fbde192cR38